test/src/test/java/test/org/dst/benchmark/core/DSTBenchmark.java

```diff
@@ -0,0 +1,77 @@
+package test.org.dst.benchmark.core;
+
+import org.dst.client.DefaultDstClient;
+import org.dst.client.DstClient;
+import test.org.dst.supplier.TestUtil;
+import java.util.ArrayList;
+import java.util.List;
+
+import static test.org.dst.benchmark.DstBenchmarkTest.benchmarkTest;
+
+public class DSTBenchmark {
+  public static final String serverAddress = "list://127.0.0.1:8082";
+
+  private int threadNum = 1;
+
+  private Runnable myTest = null;
+
+  public void setTestModule(Runnable myTest) {
+    this.myTest = myTest;
+  }
+
+  public void setThreadNum(int threadNum) {
+    this.threadNum = threadNum;
+  }
+
+  public void dstRun() {
+    try {
+      TestUtil.startRpcServer();
+      List<DstClient> clients = new ArrayList<>();
+      for (int i = 0; i < threadNum; i++) {
+        clients.add(new DefaultDstClient(serverAddress));
+      }
+      List<Thread> threads = new ArrayList<>();
+      for (int i = 0; i < clients.size(); i++) {
+        int finalI = i;
+        Thread thread =  new Thread(() -> benchmarkTest(clients.get(finalI)));
+        thread.start();
+        threads.add(thread);
+      }
+      for(int i = 0;i<threadNum;i++) {
+        threads.get(i).join();
+      }
+      TestUtil.stopRpcServer();
+    } catch (InterruptedException e) {
+      e.printStackTrace();
+    }
+  }
+
+  public void run() {
+    try {
+      TestUtil.startRpcServer();
+      if (myTest == null) {
```

Comment:
1. assert myTest is not null
2. the name `myTest` sounds too weird, could you rename it ?